scripts/mock-server.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express')
+
+
+function startMockServer(port = 3001) {
+    const app = express()
+    app.get('*', function (req, res) {
+        const { path } = req
+        const fileName = path.replace(/\//g, '-')
+        const data = require(`./mock-data/${fileName}.json`)
```

Comment:
better way to do this is through `fs.readSync(pathname)` and sending it to the client.

Reply:
Personally I don't like having require statements inside functions. only at the top level and also becomes harder if we switch to es modules.